src/NuGet.Licenses/App_Data/Licenses/Nokia-Qt-exception-1.1.json

```diff
@@ -1,7 +0,0 @@
-﻿{
```

Comment:
Why this one was deleted?

Reply:
Same as before, the script [license-text-extractor.ps1](https://github.com/NuGet/NuGet.Licenses/blob/main/src/NuGet.Licenses/license-text-extractor.ps1) did it, not me.

Reply:
But, digging into the details, the reason why it was deleted is because this license is now deprecated and the extractor script skips those https://github.com/NuGet/NuGet.Licenses/blob/main/src/NuGet.Licenses/license-text-extractor.ps1#L19.

Before my fix, the list of license exceptions was wrongly taken from https://raw.githubusercontent.com/spdx/license-list-data/45d10da0366f5fa931f60f3931fd23d5fb708de5/json/exceptions.json, where `Nokia-Qt-exception-1.1` wasn't deprecated yet, and now it is deprecated in the current/latest https://raw.githubusercontent.com/spdx/license-list-data/main/json/exceptions.json.

Reply:
Considering we assume that packages never mutate once published, and I believe it extends to embedded data like license too. What happens if the package `Nokia123` was published with the `Nokia-Qt-exception-1.1.json` license 5 years ago and the license is deleted now? Suddenly, it turns into a package without any license. Now, any enterprise consuming that package is in limbo or has to constantly check if it still has a license. Instead, regardless of what happens, the package should always keep its original license. Please correct me if I'm wrong.

Reply:
I don't disagree with you. But then, INHO, the license extractor tool should be fixed to not exclude deprecated licenses. Not my call.

Reply:
We can't focus on extractor tool right now due to high priority work we have scheduled right now. This isn't ideal and you're welcome to contribute fixing the tool.